deploy.sh

```diff
@@ -23,11 +23,15 @@ cd out
 git checkout $TARGET_BRANCH || git checkout --orphan $TARGET_BRANCH
 cd ..
 
+# Output should live in a directory with the branch name
+OUT_PATH="out/$SOURCE_BRANCH"
+
 # Clean out existing contents
-rm -rf out/* || exit 0
+rm -rf $OUT_PATH/*.html || exit 0
+mkdir $OUT_PATH || exit 0
```

Comment:
I would consider calling `mkdir` first. Sounds that `rm` doesn't fail if the path doesn't exist so it's really a preference.

Reply:
Done